source/administration/config-settings.rst

```diff
@@ -3386,7 +3431,7 @@ SQL Settings
 Enable Public Channels Materialization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. note::
-  This setting provides a fast way for System Admins to disable public channels materialization if it causes unexpected performance degradation. The feature will be enabled permanently and this setting will be removed in Mattermost v5.6, to be released on December 16, 2018.
+  This setting provided a fast way for System Admins to disable public channels materialization if it caused unexpected performance degradation in Mattermost v5.4 or v5.5. The feature was permanent and this setting was removed in Mattermost v5.6, released on December 16, 2018.
 
```

Comment:
This is a bit confusing to me.  Proposed revision to keep consistent with other removed settings:  ```*Removed in December 16, 2018 release.  The public channel materialization feature has become a permanent feature after thoroughly tested for performance.*```